packages/form/package.json

```diff
@@ -35,7 +35,7 @@
     "@hpcc-js/bundle": "^2.10.5",
     "d3-brush": "1.0.6",
     "d3-color": "1.2.3",
-    "d3-drag": "1.2.3",
+    "d3-drag": "1.2.4",
```

Comment:
Shouldn't these be made available via `@hpcc-js/common`?

Reply:
Probably - might be ok to remove - but no material change for this PR...